src/main/java/uk/gov/ons/ssdc/supporttool/schedule/FileStager.java

```diff
@@ -93,7 +93,7 @@ private JobStatus checkHeaderRow(Job job) {
             jobStatus = JobStatus.VALIDATED_TOTAL_FAILURE;
             job.setFatalErrorDescription(
                 "Header row does not match expected columns, received: ["
-                    + headerRow[index]
+                    + headerRow[index].substring(0, 150)
```

Comment:
This entire string could still end up longer than 255 characters. I think we need to increase the column width, handle these description strings more carefully, or ideally both. Happy to put this on another ticket, though.

Reply:
Realised when testing this isn't the right thing to do so I'll separate it onto another card